routers/cart-router.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+const { CartItem } = require("../models/cart-model");
+const { wrapWithTryCatch } = require("../utils");
+
+const router = express.Router();
+
+router
+  .route("/")
+  .get((req, res) => {
+    wrapWithTryCatch(res, async () => {
+      const cartItems = await CartItem.find().populate("_id");
+      const normalizedCartItems = cartItems.map((item) => {
+        const { _id, ...doc } = item._id._doc;
+        return { id: _id, ...doc, qty: item.qty };
+      });
+      res.json(normalizedCartItems);
+    });
```

Comment:
The only suggestion I would like to give is separate out all dB operations to a separate folder called controllers which will make the code even more readable.

Reply:
Thanks. That's a good suggestion. I'll do that refactoring soon